### DISCUSSION TITLE
Is there a way to exclude a directory inside a notebook?

### DISCUSSION THREAD

**Discussion by onnyyonn** [zk-org/zk#56] [Help] [answered]
I have a template directory inside my notebook. These are for some other note-taking apps, not zk templates. When I run zk, it throws some warnings for YAML syntaxes in those templates. One way to stop that would be excluding that directory from zk. Does zk support that? Or any plan to support that in future?

**Comment by mickael-menu** [comment]
It's not supported but it's a good idea, I'll take a look for the next version.

**Comment by mickael-menu** [comment, marked as answer]
@onnyyonn It's now available in [v0.6.0](https://github.com/mickael-menu/zk/releases/tag/v0.6.0)

```yaml
[note]
ignore = [
    "log-*.md"
    "drafts/*"
]
```

**Reply by tevaughan** [reply]
Might either this be best changed from "ignore" to "exclude", or might rather the command-line interface be changed from "exclude" to "ignore", in order for everything to be consistent?

**Reply by mickael-menu** [reply]
Agreed, renaming `ignore` to `exclude` in the note config would be best (keeping `ignore` working for backward compatibility). Would you open a PR with these changes?

**Reply by tevaughan** [reply]
Sure.

**Reply by tevaughan** [reply]
I opened #313
